oceanwaters/doc/setup_dev_env.md

```diff
@@ -112,8 +113,48 @@ make universalExec plexil-compiler checkpoint
 OceanWATERS.  Additional build information is available
 [here](http://plexil.sourceforge.net/wiki/index.php/Installation).
 
-* Also note that the latest version of this branch tested with OceanWATERS is tagged `OceanWATERS-v7.1`.  
+* Also note that the latest version of this branch tested with OceanWATERS is tagged `OceanWATERS-v7.1`. 
+
+### GSAP
+
+The OceanWATERS distribution includes a power system  module (`ow_power_system`) that at present
+depends on GSAP, an open-source battery prognostics executive. GSAP  must be installed
+*prior* to building the `ow_power_system` package.
+
+
+* Check out the source code:
+```
+git clone https://github.com/nasa/GSAP.git gsap
+```
+
+The default git branch is v2.0.0, which is the latest stable version
+of GSAP.  On rare occasions the latest version of this branch won't be
+compatible with OceanWATERS and you will need to check out a particular commit.
+We plan to simplify the workflow needed to keep these two software systems compatible.
+
+* Define the `GSAP_ROOT_DIR` environment variable as the location of your GSAP
+  installation, e.g.
+
+```
+export GSAP_ROOT_DIR=/home/<username>/gsap
+```
+
+NOTE: for convenience, you may wish to add the previous command to your
+shell initialization file (e.g. `.profile` or '.bashrc'), since they are needed every time
 
+* Build GSAP.
+```
+cd $GSAP_HOME
+mkdir build
+cd build
+cmake ..
+make
+```
+
+* If you have problems, see additional build information
+[here](https://github.com/nasa/GSAP/wiki).
+
+* Also note that the latest version of this branch tested with OceanWATERS is tagged `OceanWATERS-v7.1`.  
```

Comment:
This was copied from the Plexil instructions but doesn't apply to GSAP.

Reply:
Yeah meant to delete that and I think I accidentally added it twice instead, Will be removed.

Reply:
Removed